### ISSUE TITLE
Easier-to-use variable height API

### ISSUE THREAD

**Issue by dts** [tangbc/vue-virtual-scroll-list#27]
This is a great library!  Really smart architecture.

With mixed lists & complex UIs, it's not always the easiest thing to just spell out the heights of the items individually (mixing in group headers, etc).  Fortunately, your basic structure doesn't require that!  What works better for me, is to set an explicit style on each item that announces the height (i.e. `<h1 style="height: 30px">Some Header</h1>`), and then rewire the getVarSize thusly:

```
    // return a variable size (height) from given index.
    getVarSize: function (index, nocache) {
      var cache = this.delta.varCache[index]
      const get = (index) => {
        var item = this.$slots.default[index];
        if(item.data.staticStyle && item.data.staticStyle.height) {
          var m = item.data.staticStyle.height.match(/^(.*)px$/);
          return Number(m[1]);
        }
        
        return Number(item.data.attrs.height);
      }
      return (!nocache && cache && cache.size) || get(index) || 0
    },
```

If I were to push this as a PR, I'd make it so it does this if `variable` is set to `true`, and retain the original behavior if it's a function.  Thoughts?

**Comment by tangbc** [comment]
Thank you, that's a good idea! Please make this pr.

I notice that in some vue version such as [2.0.0](https://unpkg.com/vue@2.0.0/dist/vue.js), `item.data.attrs.height` is not available:
```javascript
data: {
    attrs: {
        style: 'height: 40px; line-height: 40px;'
    }
}
```
Please handle different versions of compatible processing, at least 2.0.0, and follow my code style 😀

**Comment by tangbc** [comment]
Well, let me finish it, you don't have to make pr.

**Comment by tangbc** [comment]
Released in [v1.1.9](https://github.com/tangbc/vue-virtual-scroll-list/releases/tag/v1.1.9)

**Comment by dts** [comment]
cool!  just so you know, there are two components to check: `data.staticStyle` and `data.style` - depending on how it's specified on the node.
